### DISCUSSION TITLE
Lenguajes obsoletos

### DISCUSSION THREAD

**Discussion by Idelgado1006** [hackatomik/SISCOx256#48] [Chill | Carrera]
Que lenguajes consideran que ya estan obsoletos ya sea para el mercado actual o porque hay uno que cumple su funcion mejor

**Comment by quantumquirkxyz** [comment]
En mi opinión, Smalltalk fue pionero en la programación orientada a objetos, pero ha sido en gran parte reemplazado por lenguajes más modernos como Java y C++. Estos lenguajes también implementan el paradigma orientado a objetos de manera más eficiente y con un mayor soporte en la industria, lo que hace que Smalltalk, a pesar de su importancia histórica, sea menos relevante hoy en día.

LISP y sus derivados, como Scheme y Clojure, todavía tienen un lugar en la investigación académica y en campos específicos como la inteligencia artificial. Sin embargo, en el desarrollo de software comercial, su uso ha disminuido debido a la disponibilidad de lenguajes más accesibles y con comunidades más amplias, lo que hace que LISP parezca obsoleto en este contexto.

Delphi, que alguna vez fue popular para el desarrollo de aplicaciones en Windows, ha perdido relevancia frente a lenguajes más modernos como C# y Java. Estos lenguajes ofrecen características más avanzadas y un ecosistema de desarrollo más actualizado, lo que hace que Delphi no sea la primera opción para nuevos proyectos.

Y está el Bourne Shell (sh) sigue siendo utilizado en algunos scripts de sistemas, pero ha sido en parte reemplazado por lenguajes de scripting más avanzados como Bash y Python. Estos lenguajes ofrecen una sintaxis más moderna y características adicionales que facilitan la automatización y el desarrollo, lo que reduce la prominencia de Bourne Shell.

**Reply by quantumquirkxyz** [reply]
https://youtu.be/6viIpzh7ylM?si=2nm4M1dcInW0PUQc

**Reply by quantumquirkxyz** [reply]
https://youtu.be/zuO8ma89Qvc?si=iSlU-MYrDoRlw2bj
